package.json

```diff
@@ -72,23 +74,23 @@
         "@commitlint/cli": "^18",
         "@commitlint/config-conventional": "^18",
         "@ebay/browserslist-config": "^2.7.0",
-        "@ebay/skin": "~17.1.0",
+        "@ebay/skin": "~17.2.0-next.0",
```

Comment:
Should this be changed to the actual release or will changeset do that automatically during the release?

Reply:
I'll swap it after skin release